### PR TITLE
fix: autocomplete input warning

### PR DESCRIPTION
**Pull request by zero0Halo**
**Before submitting a pull request,** please make sure the following is done:

I have done **all** of the following:

- [ ] Added a top level class to all my components `'.anchor-[COMPONENT NAME]'`.
- [x] Used [conventional commits](https://www.conventionalcommits.org) for all work.
- [ ] Tested my solution on Mobile & Tablet.
- [x] Wrote [unit tests](https://jestjs.io/docs/en/getting-started) for states and all behavior (`npm test`) and passed coverage thresholds.
- [x] Updated snapshots for all permutations (`npm test -- -u`).
- [ ] Accounted for hover, focus, blur, visited, & error states because they are not edge cases.
- [ ] Created TODOs for known edge cases.
- [x] Documented all of my changes (inline & doc site).
- [ ] Made sure that all accessibility errors are resolved.
- [x] Added [stories](https://storybook.js.org/docs/basics/introduction/) with knobs for all possible configurations.
- [x] De-linted and ran [prettier](https://github.com/prettier/prettier) (`npm run pretty`) on my code.
- [ ] Added name to OWNERS file for all new components
- [ ] If adding a new component, add its export to the rollup config
- [x] package.json version is bumped (if necessary)

---------
**Outline your feature or bug-fix below**

This fixes a very loud warning from React regarding uncontrolled components for the `Input` component, which in turn fixes the same warning from `AutoComplete` since it uses `Input`.

### CODE REVIEW

File: src/Form/Input/Input.stories.tsx [hunk 1]
```diff
@@ -18,7 +18,10 @@ const StyledStory = styled('div')`
 
 const InputStories = () => {
     const { useState } = React;
-    const [someInputVal, setSomeInputVal] = useState<string>('');
+    const [textInputVal, setTextInputVal] = useState<string>('');
```
Comment (brandonlilly) [inline src/Form/Input/Input.stories.tsx]: Whats the reasoning behind splitting this into 3 disparate states?

Comment (zero0Halo) [reply src/Form/Input/Input.stories.tsx]: That's a fair point. I'll consolidate them into a single state.

File: src/Form/Input/Input.stories.tsx [hunk 2]
```diff
@@ -16,9 +16,20 @@ const StyledStory = styled('div')`
...
+type InputState = {
+    text?: string;
+    number?: string | number;
+    tel?: string | number;
+};
+
 const InputStories = () => {
     const { useState } = React;
-    const [someInputVal, setSomeInputVal] = useState<string>('');
+    const [inputValues, setInputValues] = useState<InputState>({
```
Comment (brandonlilly) [inline src/Form/Input/Input.stories.tsx:27]: This is probably better, but was I was actually asking about is why theres a need to have different state for each of them, as opposed to all just using the same state like before.

Comment (zero0Halo) [reply src/Form/Input/Input.stories.tsx:27]: I did that at first. If they all pull from a single state value then all the form fields update at the same time with the same value. Originally only the `text` input was using state, and typing into the other fields was giving an uncontrolled form field warning.

Comment (brandonlilly) [reply src/Form/Input/Input.stories.tsx:27]: Oh derp. This is the story, not the component file. My bad, nothing to see here.

**Review by brandonlilly — APPROVED** [review]
